packages/@uppy/companion/src/server/middlewares.js

```diff
@@ -113,3 +117,18 @@ exports.cors = (options = {}) => (req, res, next) => {
     exposedHeaders: exposedHeaders.join(','),
   })(req, res, next)
 }
+
+exports.metrics = () => {
+  const metricsMiddleware = promBundle({ includeMethod: true })
+  // @ts-ignore Not in the typings, but it does exist
+  const { promClient } = metricsMiddleware
+  const { collectDefaultMetrics } = promClient
+  collectDefaultMetrics({ register: promClient.register })
+
+  // Add version as a prometheus gauge
+  const versionGauge = new promClient.Gauge({ name: 'companion_version', help: 'npm version as an integer' })
+  // @ts-ignore
+  const numberVersion = version.replace(/\D/g, '') * 1
```

Comment:
(Unrelated to the PR but maybe we can use parseInt here instead of casting with `*`, then we can also remove the `//@ts-ignore`)

Reply:
Or my personal favorite, `Number()` 😎